packages/sync/src/conflicts/strategies/strategies.ts

```diff
@@ -0,0 +1,37 @@
+import { ConflictResolutionData } from "./ConflictResolutionData";
+import { ConflictResolutionStrategy } from "./ConflictResolutionStrategy";
+
+
+// Used as default strategy for SDK
+export const clientWins: ConflictResolutionStrategy =
+  (base: ConflictResolutionData, server: ConflictResolutionData, client: ConflictResolutionData) => {
+    const { clientDiff } = calculateDiffs(base, client, server);
+    return Object.assign(base, clientDiff);
+  };
+
+function calculateDiffs(base: any, client: any, server: any) {
```

Comment:
Very rough state. We need to test it

Reply:
should the strategy just return a boolean of resolved? thats an easy way to not throw the error in the conflict link

Reply:
It works but it is not the final state.